pageserver/src/tenant/layer_map/historic_layer_coverage.rs

```diff
@@ -548,25 +626,32 @@ fn test_retroactive_simple() {
         LayerKey {
             key: 2..5,
             lsn: 105..106,
-            is_image: true,
+            is_image: false,
```

Comment:
Thanks for the fix. Seems like there wasn't also an actual bug in the implementation, because I also queried it as an image :)

Reply:
> Seems like there wasn't also an actual bug in the implementation, because I also queried it as an image :)

Exactly it only caused confusion :)